packages/stores/src/query/cosmos/supply/inflation.ts

```diff
@@ -119,9 +121,16 @@ export class ObservableQueryInflation {
         const bondedToken = new Dec(
           this._queryPool.response.data.result.bonded_tokens
         );
+
+        // community tax is normally small (~0.02), so assume zero if the query fails
+        const community_tax = new Dec(
```

Comment:
please change variable names to camel case

Reply:
+ instead of calculating community tax here, we should calculate this by creating a getter inside `ObservableQueryDistributionParams` class